skymp5-server/cpp/server_guest_lib/MpActor.cpp

```diff
@@ -49,9 +50,36 @@ void MpActor::SetEquipment(const std::string& jsonString)
 void MpActor::VisitProperties(const PropertiesVisitor& visitor,
                               VisitPropertiesMode mode)
 {
+  auto baseId = MpObjectReference::GetBaseId();
+  uint32_t raceId = GetLook() ? GetLook()->raceId : 0;
+
+  auto baseActorValues = GetBaseActorValues(baseId, raceId);
+
   MpObjectReference::VisitProperties(visitor, mode);
   if (mode == VisitPropertiesMode::All && IsRaceMenuOpen())
     visitor("isRaceMenuOpen", "true");
+
```

Comment:
These lines duplicate actor values list from `struct BaseActorValues`
Every time we modify this struct, we will have to also modify `VisitProperties` which we will likely forget to do.
Consider removing this logic duplication between files in any possible way.
My idea: Create `VisitBaseActorValues` method in `struct BaseActorValues` and move these lines into it. That's how we can at least place all code logic duplication into one file. It would be much better.